react/components/InfoCard/index.js

```diff
@@ -0,0 +1,195 @@
+import { textPositionTypes, textAlignmentTypes, callToActionModeTypes, textPostionValues, textAlignmentValues } from './SchemaTypes'
+
+import styles from './infoCard.css'
+
+const justifyTokens = {
+  [textPostionValues.LEFT]: 'justify-start',
+  [textPostionValues.CENTER]: 'justify-center',
+  [textPostionValues.RIGHT]: 'justify-end'
+}
+
+const alignTokens = {
+  [textAlignmentValues.LEFT]: 'tl',
+  [textAlignmentValues.CENTER]: 'tc',
+  [textAlignmentValues.RIGHT]: 'tr'
+}
+
+const itemsTokens = {
+  [textAlignmentValues.LEFT]: 'items-start',
+  [textAlignmentValues.CENTER]: 'items-center',
+  [textAlignmentValues.RIGHT]: 'items-end'
+}
+
+const flexOrderTokens = {
+  [textPostionValues.LEFT]: 'flex-row',
+  [textPostionValues.RIGHT]: 'flex-row-reverse'
+}
+
+const defaultValues = {
+  textPosition: textPositionTypes.TEXT_POSITION_LEFT.value,
+  textAlignment: textAlignmentTypes.TEXT_ALIGNMENT_LEFT.value,
+}
+
+const getEnumValues = enumObject => Object.values(enumObject).map(({ value }) => value)
+const getEnumNames = enumObject => Object.values(enumObject).map(({ name }) => name)
+
+const safelyGetToken = (tokenMap, valueWanted, propName) => tokenMap[valueWanted] || defaultValues[propName]
+
+class InfoCard extends PureComponent {
+  static propTypes = {
+    isFullModeStyle: bool,
+    textPosition: oneOf(getEnumValues(textPositionTypes)),
+    headline: string,
+    subline: string,
+    callToActionMode: oneOf(getEnumValues(callToActionModeTypes)),
+    callToActionText: string,
+    callToActionUrl: string,
+    imageUrl: string,
+    mobileImageUrl: string,
+    textAlignment: oneOf(getEnumValues(textAlignmentTypes)),
+  }
+
+  static defaultProps = {
+    isFullModeStyle: false,
+    textPosition: textPositionTypes.TEXT_POSITION_LEFT.value,
+    headline: null,
+    subline: null,
+    callToActionMode: callToActionModeTypes.CALL_ACTION_BUTTON.value,
+    callToActionText: '',
+    callToActionUrl: '',
+    imageUrl: '',
+    mobileImageUrl: null,
+    textAlignment: textAlignmentTypes.TEXT_ALIGNMENT_LEFT.value,
+  }
+
+  static schema = {
+    title: 'editor.info-card.title',
+    description: 'editor.info-card.description',
+    type: 'object',
+    properties: {
+      isFullModeStyle: {
+        title: 'editor.info-card.isFullMode.title',
+        description: 'editor.info-card.isFullMode.description',
+        type: 'boolean',
+        default: false,
+      },
+      textPosition: {
+        title: 'editor.info-card.textPosition.title',
+        description: 'editor.info-card.textPosition.description',
+        type: 'string',
+        enum: getEnumValues(textPositionTypes),
+        enumNames: getEnumNames(textPositionTypes),
+        default: textPositionTypes.TEXT_POSITION_LEFT.value,
+      },
+      headline: {
+        title: 'editor.info-card.headline.title',
+        description: 'editor.info-card.headline.description',
+        type: 'string',
+        default: null,
+      },
+      subline: {
```

Comment:
the text below a headline is called a `subhead`. I'm not sure "subline" exists, but I might be mistaken

Reply:
done